### DISCUSSION TITLE
Add a {{path_to_root}} variable

### DISCUSSION THREAD

**Discussion by Archie-2021** [Taitava/obsidian-shellcommands#374] [Q&A] [answered]
I want to move shell scripts used in vault portable so I want to keep them in the vault instead of my ./local/bin. But the problem I just found out is there is no way to use the path to those scripts relative to my vault. Maybe I am missing something..

**Comment by Taitava** [comment, marked as answer]
Hi! Thank you for your question! 🙂 

Does [`{{vault_path}}`](https://publish.obsidian.md/shellcommands/Variables/%7B%7Bvault_path%7D%7D) variable help?

**Reply by Archie-2021** [reply]
Yeah I think it's s a better name too.

thanks

**Reply by Taitava** [reply]
So, the variable exists already. Did it work for you? 🙂

**Comment by Archie-2021** [comment]
> So, the variable exists already. Did it work for you? 🙂

Oh :D I did suspect that is what you mean but couldn't believe it . I went into the variable section and couldn't find it (the place where you can set default value for it) and couldn't find anything related so I thought there was no such a variable.

Thanks

**Reply by Taitava** [reply]
Ah, yeah, it's easy to miss 🙂 .

**Reply by Taitava** [reply]
I moved this discussion from _Ideas_ to _Q&A_.
